addons/calendar/data/mail_template_data.xml

```diff
@@ -22,7 +22,12 @@
         Hello <t t-out="object.common_name or ''">Wood Corner</t>,<br/><br/>
 
         <t t-if="not target_responsible">
-            <t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> invited you for the <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> meeting.
+            <t t-if="object.event_id.user_id.partner_id.id == object.env.ref('base.partner_root', raise_if_not_found=False).id">
+                You have been invited by Customer to the <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> meeting.
+            </t>
+            <t t-else="">
+                <t t-out="object.event_id.user_id.partner_id.name or ''">Colleen Diaz</t> invited you for the <strong t-out="object.event_id.name or ''">Follow-up for Project proposal</strong> meeting.
```

Comment:
If the organizer removes themselves from the event, then you will have `' invited you for the name meeting'`, Maybe we could catch the case while we are are? We could include that case (`(...) or not object.event_id.user_id` in the t-if above) to have something clean :)